06_modules_lab/02.py

```diff
@@ -1,4 +1,12 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
-
+from sys import argv
+
+if len(argv) != 3:
+    print 'Usage:\n%s [a] [b]' %argv[0]

```

Comment:
Better to write the error message inside `sys.exit()`. It prints the error to stderr AND returns an error code to the operating system.